awx/main/access.py

```diff
@@ -1332,11 +1332,13 @@ def filtered_queryset(self):
     @check_superuser
```

Comment:
In the `filtered_queryset` method right above this point, there's a query using `'execution_environment_admin_role'` that needs to change to use `'member_role'`.  This is the query that gets used to see if a user can view something.